util/environment/environment.go

```diff
@@ -55,8 +55,10 @@ func toMap(s []string) map[string]string {
 				m[split[0]] = ""
 			}
 		case 2:
+			// On Windows environment variables can be mixed case, eg. Path or PATH.
+			key := strings.ToUpper(split[0])
```

Comment:
That's breaking POSIX!

Reply:
Yeah, I stumbled on the documentation. 😞